core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3205,8 +3206,15 @@ class KafkaApis(val requestChannel: RequestChannel,
           .setThrottleTimeMs(requestThrottleMs)
           .setEntries(entriesData.asJava)))
     } else {
+      val result = quotaCache.describeClientQuotas(describeClientQuotasRequest.filter())
+      val resultAsJava = new util.HashMap[ClientQuotaEntity, util.Map[String, java.lang.Double]](result.size)
```

Comment:
These conversions are annoying, but I don't know how to avoid them (unless we just use Java classes in the QuotaCache code)